daemon/info_unix.go

```diff
@@ -384,3 +393,18 @@ func Rootless(cfg *config.Config) bool {
 func noNewPrivileges(cfg *config.Config) bool {
 	return cfg.NoNewPrivileges
 }
+
+func (daemon *Daemon) runtimeStatus(cfg *configStore, runtimeName string) map[string]string {
+	m := make(map[string]string)
+	if runtimeName == "" {
+		runtimeName = cfg.Runtimes.Default
+	}
+	if features := cfg.Runtimes.Features(runtimeName); features != nil {
+		if j, err := json.Marshal(features); err == nil {
+			m["features"] = string(j)
```

Comment:
I was originally considering the features to be directly in the map, but as they're not basic key/value pairs, I guess that doesn't really work, so perhaps this is an ok alternative